src/pages/index.tsx

```diff
@@ -26,6 +27,10 @@ const meta = {
 };
 
 export default function Page() {
+  //Default platform is javascript
+  let platform = 'javascript';
```

Comment:
there is a `DEFAULT_PLATFORM` in `@/data/platforms` can we use that here instead?

Reply:
sure